api/apps/articles/model.coffee

```diff
@@ -390,7 +390,7 @@ typecastIds = (article) ->
     super_article: if article.super_article?.related_articles then _.extend article.super_article, related_articles: article.super_article.related_articles.map(ObjectId) else {}
 
 @sendArticleToSailthru = (article, cb) =>
-  cb null unless NODE_ENV is 'production'
+  return cb() unless NODE_ENV is 'production'
```

Comment:
Ha so all those errors we were seeing previously had to do with this line. I was just calling back but the rest of the method was still executing. Updated tests so we don't need to actually stub in the integration tests since they have NODE_ENV 'test'